server/controllers/mobile-phone-number-change-controller.js

```diff
@@ -6,6 +7,7 @@ import qs from "qs";
 import config from "../config.json";
 import defaultConfig from "../utils/default-config";
 import Logger from "../utils/logger";
+import {mobile_phone_number_change} from "../utils/proxyURLs";
```

Comment:
`proxyURLs` is inconsisent naming, please be consistent and change it to `proxy-urls`